pages/common/nixos-rebuild.md

```diff
@@ -0,0 +1,31 @@
+# nixos-rebuild
+
+> Reconfigure a NixOS machine.
+
+- Build and switch to the new configuration, making it the boot default:
+
+`sudo nixos-rebuild switch`
+
+- Build and switch to the new configuration, making it the boot default and naming the boot entry:
+
+`sudo nixos-rebuild switch -p {{name}}`
+
+- Build and switch to the new configuration, making it the boot default and installing updates:
+
+`sudo nixos-rebuild switch --upgrade`
+
+- Rollback changes to the configuration, switching to the previous generation:
+
+`sudo nixos-rebuild switch --rollback`
+
+- Build the new configuration and make it the boot default:
```

Comment:
See above.

Reply:
Does the change I made fix this issue? https://github.com/tldr-pages/tldr/pull/2809/commits/926f8a8c4889d5859cf8c90b5bdfcc4143a167f0